math/genvector/inc/Math/GenVector/Transform3D.h

```diff
@@ -74,13 +74,16 @@ namespace Math {
 
 */
 
+template <typename T = double>
 class Transform3D {
 
 
 public:
 
-   typedef  DisplacementVector3D<Cartesian3D<double>, DefaultCoordinateSystemTag >  Vector;
-   typedef  PositionVector3D<Cartesian3D<double>, DefaultCoordinateSystemTag >      Point;
+   typedef T Scalar;
```

Comment:
Its public. I added it to match what is done by other templated types. I actually use this in some LHCb code using the new templated class.